app/lib/homework/homework_dialog/homework_dialog.dart

```diff
@@ -959,26 +959,32 @@ class _SendNotification extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final bloc = bloc_lib.BlocProvider.of<HomeworkDialogBloc>(context);
-
     return MaxWidthConstraintBox(
       child: SafeArea(
         top: false,
         bottom: false,
-        child: _SendNotificationBase(
-          title:
-              "Kursmitglieder ${state.isEditing ? "über die Änderungen " : ""}benachrichtigen",
+        child: ListTile(
+        leading: const Icon(Icons.notifications_active),
+        title: Text(
+          "Kursmitglieder ${state.isEditing ? "über Änderungen " : ""}benachrichtigen",
+        ),
+        subtitle: state.isEditing
+            ? null
+            : const Text(
+                "Kursmitglieder über neue Hausaufgabe benachrichtigen",
+              ),
+        trailing: Switch(
```

Comment:
We can use here `Switch.adaptive` instead of `Switch`. `Switch.adaptive` uses the native platform design for the switch button, see https://api.flutter.dev/flutter/material/Switch/Switch.adaptive.html